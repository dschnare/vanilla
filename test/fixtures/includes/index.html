<html>
  <head>
    <title>Index</title>
  </head>
  <body>
    <p>This is the body!</p>
    <v:include file="./partial.html" />
    <v:include file="./widget.html" />
    <v:include file="./text.txt" />
  </body>
</html>