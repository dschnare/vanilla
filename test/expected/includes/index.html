<html>
  <head>
    <title>Index</title>
  </head>
  <body>
    <p>This is the body!</p>
    <p>This is more content added to the body.</p>
    <div class="widget">
  <p>This is a widget!</p>
</div>
    This is some text.
  </body>
</html>